fix: tickerbar , tokentable

rayanfer32 opened the pull request:
<!-- Remove the text block if it is not used or necessary -->

## Description

<!-- Please include a summary of the change or which issue is fixed. -->

-

### Type of change

- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactor ( rewrite or restructure code)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas

### Data Results

- none

### Additional Info

<!-- any additional information or context -->

- none

shrivatsabhat committed:
refactor: custom logger function to class

rayanfer32 committed:
fix: tickerbar , tokentable

rayanfer32 committed:
fix: truncation of tpken names

shrivatsabhat committed:
Merge pull request #25 from rayanfer32/refactor/logger

refactor: custom logger function to class

shrivatsabhat commented on src/components/Home/Homepage.module.scss:
```diff
@@ -1,5 +1,5 @@
 .container {
```
this is will be refactored so let it be as for now

shrivatsabhat committed:
feat: add auto label v2

shrivatsabhat committed:
ci: automate release and changelog generation

shrivatsabhat committed:
Merge pull request #29 from rayanfer32/ci/auto-label

feat: add Release Drafter v5

shrivatsabhat committed:
feat: introduce index file and views folder structure

shrivatsabhat committed:
feat: add custom link component
custom link used in alternative to next/Link

shrivatsabhat committed:
refactor: support constants as individually
destructure file structure for the constants used

shrivatsabhat committed:
feat: get string from route link

shrivatsabhat committed:
feat: support dev mode
view components while develpment time and test components

shrivatsabhat committed:
ci: update release change log template

- update release-drafter template
- restructure pull request labeler
- format and lint fix

shrivatsabhat committed:
Merge pull request #34 from rayanfer32/feat/add-dev-mode

feat: add dev mode and pullout page components to views folder

shrivatsabhat committed:
Merge branch 'develop' into ci/workflow

shrivatsabhat committed:
Merge pull request #35 from rayanfer32/ci/workflow

ci: update release draft template and pr labeler

shrivatsabhat committed:
Merge branch 'feat/styles' of https://github.com/rayanfer32/nexus-market-cap into feat/styles
